### ISSUE TITLE
Calm Business: Media & Text block background colors not displaying

### ISSUE THREAD

**Issue by thebud15** [Automattic/themes#713]
## Steps to replicate

1. Add multiple media & text blocks in the theme editor. 
2. Add an image on one side and text on the other. 
3. Give the text side a background color, such as gray and see this reflected in the editor. 
4. Publish and see how it appears on the live site.

## Result

The background colors on the media & text blocks showed up as white instead of the gray they were set to be in the editor. 

Editor view: 

![Screen Shot on 2019-04-04 at 12_53_49](https://user-images.githubusercontent.com/29046778/55588719-6430de80-56e3-11e9-9f84-aa4636824fc9.png)

Live site view:
<img width="1307" alt="Screen Shot on 2019-04-04 at 12_57_45" src="https://user-images.githubusercontent.com/29046778/55584585-63934a80-56d9-11e9-808d-4e90213cb117.png">

## Expected

The background colors of the blocks on the live site should've been gray, as they were set in the editor.

* User report: 11808765-hc 
* Follow-up ticket: 1938187-zen (The notes here also link to a Slack thread about this with some more context.)

**Comment by katiebethbrown** [comment]
Another report in 10743466-hc.

Using the Modern Business theme.

The background on the Media & Text block on the homepage is set to white, but still displaying brown on the published site.

**Comment by alaczek** [comment]
We should probably check all six variations to makes others are not affected as well.

**Comment by tijosh** [comment]
Another report of this in #2077128-zen

I was able to replicate this somewhat inconsistently on my own site using Modern Business and then Elegant Business. 

It seems to be limited to the available palette colors from the theme: 

![Screenshot](https://cld.wthms.co/rBXYVC+)

As soon as you set a custom color, the background and text colors appear on the live page as expected.

**Comment by tanjoymor** [comment]
Another case #13365111hc using Modern Business 

Modern Business and Twenty Nineteen tested on my test also. (Didn't try the custom color option noted in the last comment.)

**Comment by adelineyaw** [comment]
Another report via 2132944-zen using Modern Business as well.

Expected results: https://cld.wthms.co/VY7h09 (editor version)
Actual results: https://cld.wthms.co/1QpTjK (live site)

I've tested using Baskerville 2 and wasn't able to replicate the issue.

It looks like this CSS is overriding the Background Color Settings:

![Browser Inspect](https://cld.wthms.co/33cdee+)
Image link: https://cld.wthms.co/33cdee

Once I use the Custom Color option, the background color appears as expected:

![Background Color Settings](https://cld.wthms.co/oelR4k+)
Image link: https://cld.wthms.co/oelR4k

If you'd prefer, I can open a new issue specifically for Modern Business, just let me know 🙂

**Comment by kristarella** [comment]
I found similar results to @adelineyaw in Modern Business with this selector overridding the theme CSS:

`.entry .entry-content .wp-block-media-text:nth-child(odd)`

theme selector was:

`.entry .entry-content .has-primary-background-color`

**Comment by nagpai** [comment]
similar bug reported for Modern Business on 

#14190770-hc

Site: therugbyshed.com.au

**Comment by ktyfuller604** [comment]
Similar issue #2268929-zen - using the Modern Business theme
Site: https://pecbootcamp2019.wordpress.com/

In their case, they are on the Personal plan, so custom CSS was not an option. However, they were only trying to continue the alternating colors originally applied to multiple media/text blocks. I gave them a workaround based on my observation:

They'd added three blocks (a line, heading, and another line) the next media/text block was showing as gray instead of black. I was able to reproduce by doing the following:

1. Create a new page, and select the "team" template
2. Add a line between two media/text blocks (note the color of the block below changes each time any block is inserted between two blocks)
3. Apply a custom color; it shows in the editor, but not on the live site.

Removing one of the other blocks restored the media/text block to the black they wanted.
